### PR TITLE
Bug 1881246:  squash alerts into a single file

### PR DESCRIPTION
**Pull request by soltysh**
/assign @wking

### CODE REVIEW

File: manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ metadata:
     exclude.release.openshift.io/internal-openshift-hosted: "true"
 spec:
   groups:
-    - name: cluster-version
+    - name: kube-controller-manager
```
Comment (wking) [inline manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml:10]: Changing `spec.groups[].name` like this makes sense to me.  But the bug was about the overlap in `metadata.name`.  Can we combine the two PrometheusRule files in a single file with a single group with multiple rules?

Comment (soltysh) [reply manifests/0000_90_kube-controller-manager-operator_05_alert-pdb.yaml]: Gotcha.

**Comment by soltysh** [comment]
@wking have a look at the current state, if that's closer to your preference?

**Review by wking — APPROVED** [review]
/lgtm

**Pull request merged**